packages/openassistants/.mypy.ini

```diff
@@ -1,3 +0,0 @@
-[mypy]
-python_version = 3.10
-exclude = \.venv
```

Comment:
This seems weird, ignoring .venv is pretty typical.

Reply:
oh i replaced it to pyproject.toml, but will add this back